pandas/tests/api/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import pandas as pd
+from pandas.api.exchange.implementation import _from_dataframe
```

Comment:
prob locate in test_protocol.py for now

Reply:
My intention is to make `test_protocol.py` a verbatim copy from the spec repo, which any library would just copy-paste and add their own `conftest.py` defining the fixtures used. Hence I still want to keep them separate.